compiler/ppx/ppx_optcomp_light.ml

```diff
@@ -82,11 +80,11 @@ let get_env s =
   | None -> Sys.getenv s
   | Some p -> p
 
-let keep loc (attrs : attributes) =
+let keep loc (attrs : Ppxlib.attributes) =
```

Comment:
I would assume `attributes` to be in scope because of the `open Ppxlib.Parsetree` above

Reply:
Indeed, thanks.